### DISCUSSION TITLE
Flink-cdc 2.1.0 Oracle CDC是否支持使用Xstream的出站服务来进行数据捕获，求解

### DISCUSSION THREAD

**Discussion by peniridis** [apache/flink-cdc#722] [Q&A] [answered]
咨询一哈 Oracle CDC 支持 使用Xstream的出站服务来进行数据捕获吗，我看到docker的示例都是用logminer，但是有其他地方文章指出支持XStream。

**Comment by gtk96** [comment, marked as answer]
添加 debezium 的配置项 'database.connection.adpter' = 'xstream'， 如果使用 SQL 的方式，则在表的 option 中添加配置项 'debezium.database.connection.adpter' = 'xstream'

**Reply by peniridis** [reply]
nice 我试试
